docs/development.md

```diff
@@ -99,13 +103,24 @@ export CLI_BUCKET_NAME="cortex-cli-<your_name>"
 export CLI_BUCKET_REGION="us-west-2"
 ```
 
-Build and push all Cortex images (this will take a while)
+### Building
+
+Build and push all Cortex images (this will take a while):
 
 ```bash
 make registry-all
 ```
 
-Start Kubernetes cluster and install Cortex on it
+Build and configure the Cortex CLI:
+
+```bash
+make cli  # The binary will be placed in path/to/cortex/bin/cortex
+path/to/cortex/bin/cortex configure
```

Comment:
`cortex configure` won't work until after Cortex is installed, so this section should probably come later in the instructions